### DISCUSSION TITLE
"compiling..." info is not displayed anymore after webpack-dev-server v4 release

### DISCUSSION THREAD

**Discussion by PierBJX** [webpack/webpack#15276] [General]
With webpack-dev-server `3.x.x`, when running `webpack serve`, after a change done in the src code, it was possible to see in the console `compiling...`  and then `Compiled successfully` such like this:

![image](https://user-images.githubusercontent.com/22519163/151681881-7af890ce-26ff-420b-bf28-739e3d074851.png)

However, after the release of webpack-dev-server 4, we don't see `compiling...`  anymore, we only see `Compiled successfully`. In my opinion, it was quite useful to have `compiling...` like this we were aware that something was going on (more particularly when it is a bit long to reload).

**Comment by alexander-akait** [comment]
We remove it because it is unnecessary, why you need this? Anyway we support it on webpack-cli level, how you run dev server?

https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/plugins/CLIPlugin.js#L64

**Reply by PierBJX** [reply]
I run via a script in package.json like this:
```
  "scripts": {
    "start": "webpack serve --mode development",
  },
```

**Reply by alexander-akait** [reply]
You can use https://webpack.js.org/configuration/other-options/#level and set `log`

**Reply by PierBJX** [reply]
This option adds a lot of logs but it works. In our case, it is quite useful to have this compiling info (like before) because we have a big project and it can take some times to reload. Therefore, it can be interesting to have the information that something is going on.

**Reply by alexander-akait** [reply]
We got a lot of feedback from developers what it is very spammy, so we change it, if you can implement small plugin to achieve it, it is easy

**Reply by PierBJX** [reply]
That's right :)
